src/grantRound.ts

```diff
@@ -0,0 +1,31 @@
+import { BigInt, dataSource } from "@graphprotocol/graph-ts"
+import { AddMatchingFunds } from "../generated/templates/GrantRound/GrantRound"
+import { GrantRoundDonation } from "../generated/schema"
+
+export function handleAddMatchingFunds(event: AddMatchingFunds): void {
+    // extract the grantRounds context
+    const context = dataSource.context()
+
+    // extract the GrantRound address from the context used to create the GrantRound template
+    const address = context.getBytes("address")
+
+    // Entities can be loaded from the store using a string ID; this ID
+    // needs to be unique across all entities of the same type
+    const entity = new GrantRoundDonation(event.transaction.hash.toHex())
+
+    // Entity fields can be set based on event parameters
+    entity.id = event.transaction.hash.toHex()
+    entity.round = address
+    entity.contributor = event.params.contributor
+    entity.amount = event.params.amount
+    entity.hash = event.transaction.hash
+    entity.lastUpdatedBlockNumber = event.block.number
+    entity.lastUpdatedTimestamp = event.block.timestamp
+    entity.createdAtTimestamp = 
```

Comment:
Does it make sense to also include `entity.token`? It's arguably redundant but could be convenient to have it here? Will defer to you @gdixon on whether it's worth adding!

Reply:
We could do! I figured that address is already a part of the data we have available in the app, but it isn't anywhere in the Graphs data - so it does make sense! I'll see about getting it added later!

Reply:
(If we do add it, it would go unused in the App - so I don't think we should let this block merging/deploying these changes)